config/sync/config_ignore.settings.yml

```diff
@@ -0,0 +1,9 @@
+ignored_config_entities:
```

Comment:
uuid?

Reply:
Uuid is only for config entities. Not plain settings